### ISSUE TITLE
Option for identifier

### ISSUE THREAD

**Issue by erikng** [munki/munki-pkg#7]
As discussed yesterday, it would be nice if a preference file could be written to that added a common identifier or if you could use --identifier when create new projects.

1.
~/Library/Preferences/com.github.munki.munkipkg.plist

```
<key>identifier</key>
</string>com.test.pkg</string>
```

./munkipkg --create Test --json

`"identifier": "com.test.pkg.Test"`

2.
./munkipkg --create Test --json

`"identifier": "com.github.munki.pkg.Test"`

Desired:
./munkipkg --create Test --json --identifier "com.test.pkg"

`"identifier": "com.test.pkg.Test"`

**Comment by erikng** [comment]
Spitballing here, but perhaps a better option is for an admin to create either a plist or json (or multiple!) file that can be referenced. This current enhancement idea doesn't solve other missing/desired elements (signing key).

**Comment by gregneagle** [comment]
Sounds like maybe what you really want/need is project templates...

**Comment by erikng** [comment]
Yes, I didn't explicitly call them "templates" on my last comment, but that's exactly what I mean.

**Comment by gregneagle** [comment]
munkipkg is being retired. Consider using https://github.com/codecarton/swiftpkg instead.
